src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationInvalidArrayInitIndent.java

```diff
@@ -131,6 +131,6 @@ private void myFunc3()
             "hello", "chekstyle"
             }
         )
-      void worked();
+        void worked();
```

Comment:
please revert this. You can edit the log message inside tests but can't the test inputs itself.

Reply:
Done